packages/client-core/src/user/components/UserMenu/index.tsx

```diff
@@ -29,10 +29,11 @@ import React from 'react'
 import { getState, useMutableState } from '@ir-engine/hyperflux'
 import IconButtonWithTooltip from '@ir-engine/ui/src/primitives/mui/IconButtonWithTooltip'
 
+import PopupMenu from '@ir-engine/ui/src/primitives/tailwind/PopupMenu'
 import { AppState } from '../../../common/services/AppService'
 import { useShelfStyles } from '../../../components/Shelves/useShelfStyles'
-import styles from './index.module.scss'
 import { PopupMenuServices, PopupMenuState } from './PopupMenuService'
+import styles from './index.module.scss'
```

Comment:
we should use tailwind everywhere. hence, we should not introduce css styles.

this can also introduce caveats like the one below.
![image](https://github.com/user-attachments/assets/9083bf7c-7a9d-4fb1-9756-6e3d391b6067)

(we should fully use tailwind and discard mui)

Reply:
it's just being used to add stars to the background, nothing related to MUI

Reply:
lets add this to tailwind's [components.css](https://github.com/ir-engine/ir-engine/blob/ae73366ddc58b3ccef934220a4ad2c707d8397d1/packages/client/src/themes/components.css)

we will be removing the module css files at a later point

Reply:
Done